dir.targets

```diff
@@ -41,12 +41,18 @@
   <Target Name="Test" />
 
   <Target Name="EnsureDependenciesDeployed"
-          BeforeTargets="Build">
+          AfterTargets="_RestoreBuildToolsWrapper">
```

Comment:
On Linux we download a package as part of the Build Tools (Microsoft.DotNet.BuildTools) which contains the PrereleaseResolveNuGetPackageAssets task used when deploying dependencies. This makes sure we run it after Build Tools are restored.